addon/components/ember-table-base-cell.js

```diff
@@ -39,7 +40,14 @@ export default class EmberTableCell extends Component {
 
   scheduleSync() {
     scheduler.schedule('sync', () => {
-      const { height } = this.element.getBoundingClientRect();
+      let { height } = this.element.getBoundingClientRect();
+
+      if (this.get('isFixed')) {
```

Comment:
We should revert this commit until we're ready to fully address the issues of keeping the row height and cell height in sync. In the meantime, we can opt into the performance improvement of using fixed heights.